stdlib/3/ast.pyi

```diff
@@ -23,9 +24,18 @@ class NodeTransformer(NodeVisitor):
 _T = TypeVar('_T', bound=AST)
 
 if sys.version_info >= (3, 8):
+    @overload
+    def parse(source: Union[str, bytes], filename: Union[str, bytes] = ..., mode: Literal["exec"] = ...,
+              type_comments: bool = ..., feature_version: int = ...) -> Module: ...
```

Comment:
Optional: You could also add the following overload, since `exec` is the default mode:

```python
    @overload
    def parse(source: Union[str, bytes], filename: Union[str, bytes] = ..., *,
              type_comments: bool = ..., feature_version: int = ...) -> Module: ...
```

Similar for the Python 3.7 version.

(Also a style nit: Please don't add empty lines between overloads.)

Reply:
Sorry I didn't get back to this sooner, thanks for the merge! Regarding this comment, I tried adding that overload and IIRC mypy complained about overlapping overloads. It seems that implicitly due to ordering of overloads, mypy already treats this as the "default" one. (I think the clearer way to express this to typecheckers would be to actually include the default value of the optional argument in the stub).